src/plasoscaffolder/bll/mappings/init_mapping.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+""" Module representing function for the diffrent files """
+from plasoscaffolder.bll.mappings.mapping_helper import render_template
+
+def get_formatter_init_create(plugin_name: str) -> str:
+  """
+  renders formatter init if you want to create new init file
+
+  :param plugin_name: the plugin name
```

Comment:
Please use "Google style" docstrings, per the Plaso style guide. You can set this up in pycharm per these instructions: https://www.jetbrains.com/help/pycharm/2016.3/python-integrated-tools.html